kernel/kernel-api/src/test/scala/io/delta/kernel/TransactionSuite.scala

```diff
@@ -156,7 +156,7 @@ class TransactionSuite extends AnyFunSuite with VectorTestUtils with MockEngineU
         VectorUtils.buildArrayValue(Seq.empty.asJava, StringType.STRING),
         Optional.empty(),
         stringStringMapValue(configMap.asJava))
-      val txnState = TransactionStateRow.of(metadata, "table path")
+      val txnState = TransactionStateRow.of(metadata, "table path", 200)
```

Comment:
This seems like a magic number used in two different places. Can we define this as a static constant somewhere?

Reply:
The number doesn't really matter here it's just to avoid compile error. Would a comment annotating the magic number help? `/* maxRetries */`